proto/logger.proto

```diff
@@ -4,32 +4,41 @@ package logger;
 import "google/protobuf/timestamp.proto";
 
 service Logger {
+  // Store logs
+  rpc StoreLogs(StoreLogsRequest) returns (StoreLogsResponse);
+
   // Get stored logs
   rpc GetLogs(LogsRequest) returns (LogsResponse);
 
   // Get fresh logs as they are incoming
-  rpc GetLogsStream(LogsRequest) returns (stream LogItem);
+  rpc GetLogsStream(LogsRequest) returns (stream FetchedLogItem);
+}
+
+message StoreLogsRequest {
+  repeated StoredLogItem logs = 1;
+}
+
+message StoreLogsResponse {
+  bool success = 1;
 }
 
 message LogsRequest {
   string deployment_id = 1;
 }
 
 message LogsResponse {
-  repeated LogItem log_items = 1;
+  repeated FetchedLogItem log_items = 1;
 }
 
-message LogItem {
+message StoredLogItem {
   string service_name = 1;
-  google.protobuf.Timestamp timestamp = 2;
-  LogLevel level = 3;
-  bytes fields = 4;
+  string deployment_id = 2;
+  google.protobuf.Timestamp tx_timestamp = 3;
+  bytes data = 4;
 }
 
-enum LogLevel {
-  Trace = 0;
-  Debug = 1;
-  Info = 2;
-  Warn = 3;
-  Error = 4;
+message FetchedLogItem {
+  string service_name = 1;
+  google.protobuf.Timestamp tx_timestamp = 2;
+  bytes data = 3;
```

Comment:
How bout this structure? (feel free to change the names)
```rs
message LogItem {
  string deployment_id = 1;
  LogLine log_line = 2;
}

message LogLine {
  string internal_origin = 1;
  google.protobuf.Timestamp timestamp = 2;
  string line = 3;
}

```

Reply:
Done 👍

Reply:
Upsy, I have some duplicates. Will fix that.